airflow/cli/commands/worker_command.py

```diff
@@ -76,17 +85,19 @@ def worker(args):
             stderr=stderr,
         )
         with ctx:
-            sub_proc = subprocess.Popen(['airflow', 'serve_logs'], env=env, close_fds=True)
+            if skip_serve_logs is False:
```

Comment:
Nit: Let's factor this out into a private function _serve_logs() to dedup (I know the previous logic didn't have it but we are slightly increasing complexity with this change anyways so I think we should fix this).

Reply:
done